2022/24xxx/CVE-2022-24387.json

```diff
@@ -1,18 +1,100 @@
 {
```

Comment:
{} is not allowed for {'reference_data': [{'name': 'https://csirt.divd.nl/DIVD-2021-00029', 'refsource': 'CONFIRM', 'url': 'https://csirt.divd.nl/DIVD-2021-00029'}, {'name': 'https://csrit.divd.nl/CVE-2022-24387', 'refsource': 'CONFIRM', 'url': 'https://csrit.divd.nl/CVE-2022-24387'}]}